### PR TITLE
Second audit changes

### PR DESCRIPTION
**Pull request by adamdossa**

### CODE REVIEW

**Comment by adamdossa** [comment]
Closes #152

File: contracts/SecurityToken.sol [hunk 1]
```diff
@@ -346,13 +345,13 @@ contract SecurityToken is ISecurityToken, IERC20 {
      * @return bool success
      */
     function withdrawPoly() public returns (bool success) {
-        require(hasOfferingStarted);
```
Comment (pabloruiz55) [inline contracts/SecurityToken.sol:349]: Why has this require been taken out?

File: contracts/SecurityToken.sol [hunk 2]
```diff
@@ -361,19 +360,19 @@ contract SecurityToken is ISecurityToken, IERC20 {
      * @return bool success
      */
     function voteToFreeze(address _recipient) public onlyShareholder returns (bool success) {
-        require(delegate != address(0));
-        require(hasOfferingStarted);
```
Comment (pabloruiz55) [inline contracts/SecurityToken.sol:365]: Why has this require been taken out?

Comment (satyamakgec) [reply contracts/SecurityToken.sol:365]: I think as per the audit report, we don't need another variable hasOfferingStarted to check whether the offering contract address set or not. we can do by `require(offering != address(0))`

Comment (adamdossa) [reply contracts/SecurityToken.sol:365]: Yep - looked like it was functionally equivalent to me.

**Review by satyamakgec — APPROVED** [review]

**Comment by satyamakgec** [comment]
@pabloruiz55 I approved the PR it looks fine to me. If you have any insight please let me know otherwise I will merge it

File: contracts/SecurityToken.sol (hunk 1 above)
Comment (sangheraio) [reply contracts/SecurityToken.sol:349]: it checks offering != 0x0 instead, accomplishes the same thing

**Pull request merged**